src/main.rs

```diff
@@ -444,35 +427,34 @@ fn cmd_list(lists: &[TodoList], name: &str) -> CmdResult {
     }
 }
 
-fn cmd_lists(lists: &[TodoList]) -> CmdResult {
+fn cmd_lists(lists: &[TodoList]) -> (std::string::String, bool) {
```

Comment:
minorist nitpick (because i can muahaha) that it should probably be `(String, bool)` rather than `std::string::String` and also I kinda prefer it to still be a `CmdResult` even though it doesn't technically need to be just for consistency but idk tbh could leave it

Reply:
alright, will fix! the namespace path was auto-inserted by rust-analyser when i made the fix and i didn't notice it :P